rasa/nlu/featurizers/dense_featurizer/dense_featurizer.py

```diff
@@ -0,0 +1,56 @@
+from rasa.shared.exceptions import InvalidConfigException
+from typing import Text
+import numpy as np
+
+from rasa.nlu.featurizers.featurizer import Featurizer2
+from rasa.utils.tensorflow.constants import MEAN_POOLING, MAX_POOLING
+
+
+class DenseFeaturizer2(Featurizer2[np.ndarray]):
```

Comment:
as it doesn't implement all abstract methods?
```suggestion
class DenseFeaturizer2(Featurizer2[np.ndarray], abc.ABC):
```